frontend/public/style/_overrides.scss

```diff
@@ -71,13 +71,8 @@ form.pf-c-form {
   font-family: var(--pf-global--FontFamily--sans-serif);
 }
 
-.pf-c-breadcrumb {
-  padding-bottom: 12px;
-  padding-top: 25px;
-
-  &--no-padding-top {
-    padding-top: 0;
-  }
+.co-breadcrumb {
```

Comment:
This rule is no longer an override, so it should move out of this file to `_common.scss`.

Reply:
moved to `_common.scss`